application/README.md.tt

```diff
@@ -0,0 +1,16 @@
+[![Lint](https://github.com/jetrockets/rails-template/actions/workflows/lint.yml/badge.svg)](https://github.com/jetrockets/rails-template/actions/workflows/lint.yml)
+# <%= app_name %>
+
+## Description
+
+This application was generated from the recommended template for Rails 7 applications by JetRockets
+
+## Requirements
+
+This application currently works with:
```

Comment:
Lets rephrase this with something like "Below is a list (not complete) of libraries that are included and configured:"